sidebars.js

```diff
@@ -144,6 +144,13 @@ module.exports = {
         type: 'category',
         label: 'Features',
         items: [
+          {
+            type: 'category',
+            label: 'Browser',
+            items:[
+              'js/features/browser/sdk-in-browser'
+            ]
+          },
```

Comment:
Do you think we'll have more browser dedicated materials to make it a category?

Reply:
Hm, actually no. But it was in a browser folder. I would move it to the guides sections?